### ISSUE TITLE
"Load GDScript documentations failed" after extension auto updated to 0.3.5

### ISSUE THREAD

**Issue by Cryszon** [GodotExplorer/godot-tools#52]
Looks like there was a new version of the extension published to the Visual Studio Code Marketplace that causes at least the syntax highlighting to fail with a "Load GDScript documentations failed" error.

In the Marketplace the extension has the folloving in the More Info section:

Version | 0.3.5
-- | --
Last updated | 1/30/2018, 4:50:01 PM

In addition the Change Log in the Marketplace displays the 0.3.5 version with following changes:
- Add option to disable syntax checking for GDScript
- Improved inline if else statement syntax checking
- More resource type supported for syntax highglight
- Bump default godot version to 3.0
- Sync the documentations from godot 3.0

Latest commit to this repository however was on Jan 1 and there's no mention of 0.3.5 anywhere.

**Comment by Geequlim** [comment]
What is the godot version of your project?

**Comment by Geequlim** [comment]
> Latest commit to this repository however was on Jan 1 and there's no mention of 0.3.5 anywhere.

I forgot to push the commit here.

**Comment by Cryszon** [comment]
Ok. So I manually removed 0.3.5 then installed 0.3.4, which worked fine. Then I re-enabled auto-updating which updated the extension back to 0.3.5 and now 0.3.5 seems to be working. I removed the leftover 0.3.4 version from the extensions directory and then removed and reinstalled 0.3.5 again and it still seems to be working. I'm running Godot 2.1.4, but I suspect there might have been an issue when the extension automatically updated.
